subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -233,6 +226,19 @@ class SubscriptionService(
             .map { it.t1.size }
     }
 
+    private fun prepareUpdateSubOnId(attributeName: String, update: Update, id: String): Mono<Int> {
```

Comment:
Maybe better to use this signature updateSubscriptionAttribute(subscriptionId, attributeName, updateStatement)